### DISCUSSION TITLE
[Add domain] Smart TV blocklist feedback

### DISCUSSION THREAD

**Discussion by mkleinschnitker** [blocklistproject/Lists#227] [General]
Hi, 

thanks for adding a new list. It works properly, but I'd would add the following domains:

- osb-apps-v2.samsungqbe.com
- osb.samsungqbe.com
- gld.push.samsungosp.com

_Originally posted by @blocklistproject in https://github.com/blocklistproject/Lists/issues/219#issuecomment-793998745_

**Comment by blocklistproject** [comment]
I have pushed this update. Thank you

**Reply by mkleinschnitker** [reply]
Thanks.

**Comment by rh-fcristin** [comment]
osb-apps-v2.samsungqbe.com record breaks the app store on newer samsung smart TV, tested on Q60 and QN90 series (both on latest fw). It possibly break also initial TV configuration along with osb-v2.samsungqbe.com record, my initial issue was found when reinstalling of the Q60 via smartthings android app, the procedure was failing. After unlocking both records it works flawlessly.
